webpack.config.js

```diff
@@ -0,0 +1,64 @@
+const path = require('path');
+const Dotenv = require('dotenv-webpack');
+const Webpack = require('webpack');
+const HTMLWebpackPlugin = require('html-webpack-plugin');
+const MiniCssExtractPlugin = require('mini-css-extract-plugin');
+
+module.exports = {
+  mode: process.env.mode,
+  entry: path.join(__dirname, 'src', 'index.js'),
+  output: {
+    path: path.join(__dirname, 'dist'),
+    filename: 'index.js',
+    publicPath: '/',
+  },
+  resolve: { extensions: ['.js', '.jsx'], modules: ['node_modules'] },
+  stats: 'errors-only',
+  module: {
```

Comment:
Also, let's say we JSON data, will our configurations be sufficient?

Reply:
I didn't get your question well? `If we have JSON data?` Like where?